src/map/itemdb.hpp

```diff
@@ -755,11 +755,18 @@ enum e_itemshop_restrictions {
 enum e_item_drop_effect : uint16 {
 	DROPEFFECT_NONE = 0,
 	DROPEFFECT_CLIENT,
+#if PACKETVER < 20200304
 	DROPEFFECT_WHITE_PILLAR,
+#else
+	DROPEFFECT_WHITE_PILLAR = 1,	// White pillar was removed from 2020-03-04+
+#endif
```

Comment:
The one you're actually overwriting is DROPEFFECT_CLIENT, not DROPEFFECT_WHITE_PILLAR it should be

```suggestion
#if PACKETVER < 20200304
	DROPEFFECT_WHITE_PILLAR,	// White pillar was removed from 2020-03-04+
#endif
```

Reply:
This will cause all the other constants below it to shift by 1. Is this what happened on kRO or did they just stop using that white pillar value?

Reply:
@aleos89 I didn`t found white pillar inside client, and if you check gif in issue you can see that all colors are wrong.
Need to shift it by 1.
![99214528-f3c8c980-2802-11eb-950e-a3987f36c29d](https://user-images.githubusercontent.com/16213511/99672081-fb4ad580-2a83-11eb-9dac-62e18445aaa5.gif)
> - Id: 501
AegisName: Red_Potion
Name: Red Potion
Type: Healing
Buy: 10
Weight: 70
Flags:
DropEffect: **WHITE_PILLAR**
Script: |
itemheal rand(45,65),0;